tests/storage/zfsvol/conftest.py

```diff
@@ -8,6 +8,7 @@
 def host_with_zfsvol(host_with_saved_yum_state):
     host = host_with_saved_yum_state
     host.yum_install(['xcp-ng-xapi-storage-volume-zfsvol'])
+    host.restart_toolstack(verify=True)
```

Comment:
we should also maybe add `systemctl daemon-reload` ?

Reply:
@stormi Can we directly add a `systemctl daemon-reload`  in the spec file of the smapi v3? Is there any contraindication to this?

Reply:
Any RPM which updates a service file should usually call it, there are even macros that do it for you.